integration/tests/cook/test_multi_user.py

```diff
@@ -62,7 +62,7 @@ def test_multi_user_usage(self):
             for i, user in enumerate(users):
                 with user:
                     for j in range(i):
-                        job_uuid, resp = util.submit_job(self.cook_url, command='sleep 240', **job_resources)
+                        job_uuid, resp = util.submit_job(self.cook_url, command='sleep 480', **job_resources)
```

Comment:
Why did this change?

Reply:
Was getting timeouts (same answer for all 3)